.circleci/config.yml

```diff
@@ -88,8 +90,8 @@ jobs:
     steps:
       - checkout
       - config-path
-      - restore_cache:
-          key: venv-v2-{{ checksum "dev-requirements.txt" }}
+#      - restore_cache:
+#          key: venv-v2-{{ checksum "dev-requirements.txt" }}
```

Comment:
why?

Reply:
I want to check that the install works on python 3.8 (it does not) so I need to not use cache to force the installation.
These PR are just tests for now, I won't merge them, but if I leave them as draft circleci won't run them I believe.